ckanext/switzerland/helpers/date_helpers.py

```diff
@@ -167,10 +167,10 @@ def display_if_other_formats(value):
 def transform_any_date_to_isodate(date_field):
     """Transform any stored date format into an isodate.
     """
-    isodate_field = store_if_date_picker_date(date_field)
+    isodate_field = store_if_isodate(date_field)
```

Comment:
we have so many `if`s here, would it make sense to rewrite this function with something like that:
#put all functions we are using for transformation in dict/list and then loop through it and change date format


`format_functions = {
    'isodate': store_if_isodate,
    'date_picker_date': store_if_date_picker_date,
    'timestamp': store_if_timestamp,
    'other_formats': store_if_other_formats
}

for format_name, format_function in format_functions.items():
    isodate_field = format_function(date_field)
    if isodate_field:
        return isodate_field`